RadixWallet/Clients/AccountPortfoliosClient/AccountPortfoliosClient+Live.swift

```diff
@@ -133,7 +133,7 @@ extension AccountPortfoliosClient: DependencyKey {
 				cacheClient.removeFolder(.onLedgerEntity(.account(accountAddress.asGeneral)))
 			}
 
-			let account = try await onLedgerEntitiesClient.getAccount(accountAddress)
+			let account = try await onLedgerEntitiesClient.getAccount(accountAddress).nonEmptyVaults
```

Comment:
This was causing users to see tokens with zero values when they did performa pull to refresh in Account details

Reply:
If this is done here, will that not affect all the endpoints on the client? In transaction history I need the full portfolio, including empty vaults.

Reply:
right should be able to do it later, in view layer is Gustafs meaning?

Reply:
probably will it will,
will move this at UI level